programs/account-compression/Cargo.toml

```diff
@@ -14,7 +14,9 @@ no-entrypoint = []
 no-idl = []
 no-log-ix-name = []
 cpi = ["no-entrypoint"]
-default = []
+custom-heap = ["light-heap"]
+mem-profiling = []
+default = ["custom-heap", "mem-profiling"]
```

Comment:
maybe we can put mem-profiling into test-sbf to only execute it then?

Reply:
Yes, that should work. Good idea.